### PR TITLE
[Media upload] Add media pickers to the Room screen and the composer

### LINKED ISSUE #360: [Media Upload] Implement pickers
We need to be able to allow the user to pick media from their gallery, files or directly from the camera.

This needs to be done through different system pickers and grouped together in a nice, reusable, interface.

### PR DESCRIPTION
**Pull request by jmartinesp**
## Changes

- Modifies `ModalBottomSheetLayout` to match designs.
- Attaches the different media pickers to the room and composer components in `MessagesView` and `MessageComposerPresenter`.
- Works around a bug where the keyboard wouldn't be hidden when calling `focusManager.clearFocus()` because the focused component is inside an `AndroidView`.
- Remove translations moved to a different module.
- Modify localazy script to be able to exclude regexes too (`screen_room_*` clashes with `screen_room_details_*` and `screen_room_member_details_*`).

## Why

Closes #360 .

## Tests

Get inside a room, play with the different pickers. Right now they don't do anything with the retrieved media, that should be handled in future PRs.

### CODE REVIEW

File: tools/localazy/generateLocalazyConfig.py [hunk 1]
```diff
@@ -36,10 +36,13 @@ def convertModuleToPath(name):
...
 for entry in config["modules"]:
     # Create action for the default language
+    excludeRegex = regexToAlwaysExclude
+    if "excludeRegex" in entry:
+        excludeRegex += entry["excludeRegex"]
     action = baseAction | {
         "output": convertModuleToPath(entry["name"]) + "/src/main/res/values/localazy.xml",
         "includeKeys": list(map(lambda i: "REGEX:" + i, entry["includeRegex"])),
-        "excludeKeys": list(map(lambda i: "REGEX:" + i, regexToAlwaysExclude)),
+        "excludeKeys": list(map(lambda i: "REGEX:" + i, excludeRegex)),
```
Comment (bmarty) [inline tools/localazy/generateLocalazyConfig.py:45]: Thanks for the enhancement. You also have to update the action for the translations, [this line on develop](https://github.com/vector-im/element-x-android/blob/develop/tools/localazy/generateLocalazyConfig.py#L54):
From
```
            "excludeKeys": list(map(lambda i: "REGEX:" + i, regexToAlwaysExclude)),
```
to
```
            "excludeKeys": list(map(lambda i: "REGEX:" + i, excludeRegex)),
```

Comment (bmarty) [reply tools/localazy/generateLocalazyConfig.py:45]: Why do you compute again `excludeRegex`?

Comment (jmartinesp) [reply tools/localazy/generateLocalazyConfig.py:45]: 🤦  I don't know why I thought they were 2 different functions. It should be fixed now.

File: features/messages/impl/src/main/res/values/localazy.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
+  <string name="screen_room_attachment_source_camera">"Camera"</string>
+  <string name="screen_room_attachment_source_camera_photo">"Take photo"</string>
+  <string name="screen_room_attachment_source_camera_video">"Record a video"</string>
+  <string name="screen_room_attachment_source_files">"Attachment"</string>
+  <string name="screen_room_attachment_source_gallery">"Photo &amp; Video Library"</string>
+  <string name="screen_room_error_failed_processing_media">"Failed processing media to upload, please try again."</string>
+</resources>
```
Comment (bmarty) [inline features/messages/impl/src/main/res/values/localazy.xml:9]: I would not use `screen_room_` prefix for those string, maybe something more generic like `screen_attachment_picker_`. And move to a new module, see my other remark.

**Review by bmarty — APPROVED** [review]
Nice work! A few non-blocking remarks.

File: features/logout/api/src/main/res/values/localazy.xml [hunk 3]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
   <string name="screen_signout_confirmation_dialog_content">"Are you sure you want to sign out?"</string>
-  <string name="screen_signout_confirmation_dialog_submit">"Sign out"</string>
   <string name="screen_signout_confirmation_dialog_title">"Sign out"</string>
   <string name="screen_signout_in_progress_dialog_content">"Signing out…"</string>
+  <string name="screen_signout_confirmation_dialog_submit">"Sign out"</string>
```
[the remark this answers is not in the document]
Comment (jmartinesp) [reply features/logout/api/src/main/res/values/localazy.xml:6]: It was actually automatic. I also found it quite weird.

File: features/messages/impl/src/main/res/values/localazy.xml (hunk 2 above)
Comment (jmartinesp) [reply features/messages/impl/src/main/res/values/localazy.xml:9]: I wanted to use some `common_*` strings for these, but we agreed with the iOS team to use `screen_room_` and then duplicate & link the strings if needed in Localazy.

**Pull request merged**